localisation/00_zuwc_united_localisation.csv

```diff
@@ -620,24 +620,24 @@ EVTOPTA_WCFEL_1010_case_01;After many trials and tribulations, Azeroth has been
+EVTDESC_WCBDP_60_horde;The expedition of the Horde passed through the shroud of the Dark Portal and is ready to gain a foothold in this new wild world. Our once beautiful world now resembles an immense wasteland. Without food. Without water. Without life. Nevertheless, here you can still survive...\nHere, among the red sands, stands the Hellfire Citadel, our very first bastion against invaders from the world of Azeroth. Now it belongs to the orcs of Ner'zhul, and the sorcerer himself tries to open portals to other worlds.\nThe main attack on the fortress seems insane to many. The elders propose to take a small fortress near the portal and acquire a fleet to strike at the enemy from the flanks. This can ease the battle to come in the future.;;Die Expeditionsstreitmacht der Horde durchquerte den Schleier des dunklen Portals und ist bereit, in dieser neuen, wilden Welt Fu� zu fassen. Unsere einst sch�ne Welt gleicht jetzt einem riesigen �dland - ohne Essen, ausgetrocknet und leblos. Trotzdem kann man hier noch �berleben...\nHier steht inmitten des roten Sandes die H�llenfeuerzitadelle, unsere erste Bastion in der Welt von Azeroth. Jetzt liegt es bei Ner�zhuls Orks, weitere Portale in andere Welten zu �ffnen.\nDer Frontalangriff auf die Festung erscheint vielen verr�ckt. Die �ltesten schlagen vor, eine kleine Festung in der N�he des Portals zu errichten und eine Flotte zu bauen, um dem Feind in die Flanke zu fallen. Dies k�nnte die bevorstehende Schlacht zu unseren Gunsten wenden.;;;;;;;;;;;x
```

Comment:
"\nHier steht inmitten des roten Sandes die Höllenfeuerzitadelle, unsere erste Bastion gegen Eindringlinge aus der Welt von Azeroth."

Reply:
Thank you, will change.